modules/sdk/project-templates/project-templates-theme-contributor/src/main/resources/archetype-resources/pom.xml

```diff
@@ -78,4 +83,12 @@
 			</plugin>
 		</plugins>
 	</build>
+	<dependencies>
+		<dependency>## ignore dependency comparison
```

Comment:
Please remove comment - `## ignore dependency comparison`

Reply:
@petershin This is to pass `ProjectTemplatesFilesTest._testPomXML` It checks the dependencies and compares them between maven and gradle. We need a way to ignore the dependency counter in maven because the dependencies do not exist in gradle.

Reply:
Like Lawerence said, we need a way to signal the `ProjectTemplateFilesTest` test to not compare it with gradle since there is no gradle analog for this.